src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/ClassRoomRouter.tsx

```diff
@@ -22,14 +22,14 @@ const ClassRoomRouter = () => {
       <Box pad="medium">
         <Switch>
           <Route path={classroomCreatePath} exact>
```

Comment:
The refacto is bigger but we should also rename `create` route in `manage`. WDYT ?

Reply:
The route create (`/my-contents/classroom/create`) open the create modal, IMO we should keep it like that, it is more accurate.

Reply:
You are right !